topi/python/topi/arm_cpu/conv2d.py

```diff
@@ -171,14 +168,21 @@ def _decl_spatial_pack(cfg, data, kernel, strides, padding, dilation, layout, ou
     VH = cfg["tile_oh"].size[-1]
     VW = cfg["tile_ow"].size[-1]
 
-    dvshape = (N, OH // VH, OW // VW, CI, VH*HSTR + KH-1, VW*WSTR + KW-1)
     kvshape = (CO // VC, CI, KH, KW, VC)
     ovshape = (N, CO // VC, OH // VH, OW // VW, VH, VW, VC)
     oshape = (N, CO, OH, OW)
 
-    data_vec = tvm.compute(dvshape, lambda n, h, w, ci, vh, vw:
-                           data_pad[n][ci][h*VH*HSTR+vh][w*VW*WSTR+vw],
-                           name='data_vec')
+    if dilation_h != 1 or dilation_w != 1:
+        dvshape = (N, OH // VH, OW // VW, CI, VH, VW, KH, KW)
```

Comment:
This packing strategy does not make sense to me.
We should keep the 6-dimension packing strategy.

Reply:
updated.

Reply:
Sorry, now I think `(N, OH // VH, OW // VW, CI, KH, KW, VH, VW)` will be better. Since our loop order is `kh, kw, vh, vw`, this layout has the best locality.

Reply:
Thanks for your suggestion. Updated.